modules/virt-importing-vm.adoc

```diff
@@ -0,0 +1,121 @@
+// * virt/virtual_machines/importing_vms/virt-importing-rhv-vm.adoc
+[id='virt-importing-vm_{context}']
+ifdef::virt-importing-vmware-vm[]
+= Importing a VMware virtual machine or template
+
+You can import a VMware virtual machine or template using the virtual machine wizard.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+= Importing a Red Hat Virtualization virtual machine
+
+You can import a Red Hat Virtualization (RHV) virtual machine using the virtual machine wizard.
+endif::[]
+
+.Procedure
+
+. In the web console, click *Workloads* -> *Virtual Machines*.
+. Click *Create Virtual Machine* and select *Import with Wizard*.
+
+ifdef::virt-importing-vmware-vm[]
+. In the *Connect to Provider* screen, select *VMware* from the *Provider* list.
+. Select *Connect to New Instance* or a saved vCenter instance from the *vCenter instance* list.
+
+* If you select *Connect to New Instance*, enter the *vCenter hostname*, *Username*, and *Password*.
+* If you select a saved vCenter instance, the wizard connects to the vCenter instance using the saved credentials.
+endif::[]
+ifdef::virt-importing-rhv-vm[]
+. In the *Connect to Provider* screen, select *Red Hat Virtualization (RHV)* from the *Provider* list.
+. Select *Connect to New Instance* or a saved RHV instance from the *RHV Instance* list.
+
+* If you select *Connect to New Instance*, fill in the following fields:
+
+** *API URL*: For example, `\https://<RHV_Manager_FQDN>:8443/ovirt-engine/api`
+** *CA certificate*: Click *Browse* to upload the RHV Manager CA certificate or paste the CA certificate into the field.
++
+You can obtain the RHV Manager CA certificate with the following command:
++
+----
+$ curl \
+--output ca.crt \
+'http://<RHV_Manager_FQDN>/ovirt-engine/services/pki-resource?resource=ca-certificate&format=X509-PEM-CA'
+----
+
+** *Username*: RHV Manager user name in the following format: `<username>@<domain>`
+** *Password*: RHV Manager password
+
+* If you select a saved RHV instance, the wizard connects to the RHV instance using the saved credentials.
+endif::[]
+
+. Click *Check and Save* and wait for the connection to complete.
+
+ifdef::virt-importing-rhv-vm[]
+. Select a cluster and a virtual machine to import.
+endif::[]
+ifdef::virt-importing-vmware-vm[]
+. Select a virtual machine or a template to import.
+endif::[]
+
+. Click *Next*.
+. In the *Review* screen, review your settings.
+
+ifdef::virt-importing-rhv-vm[]
+. Optional: You can select *Start virtual machine on creation*.
+endif::[]
+
+. Optional: Click *Edit* to update the following settings:
+
+* *General*:
+
+ifdef::virt-importing-rhv-vm[]
+** *Name*: You must update the virtual machine name if the same name is being used by another virtual machine in the same namespace.
+** *Description*: Optional
+
+* *Storage* > *Storage Class*: If you do not select a storage class, {VirtProductName} uses the default storage class to create the virtual machine.
```

Comment:
not sure if we should mention that it selects a PV with no class if default storage class is not available?

Reply:
So what happens if a storage class is not selected? 

If there is a default storage class, is that what the VM gets?

If there is no default storage class, does the VM disk have no storage class?

Reply:
yes

```
if disk with no SC specified
    if default SC exists then
        bind disk (PVC) to PV with default SC
    else
        bind disk (PVC) to PV with no SC specified
```